packages/pharos-cli/src/utils/initComponentsUtil.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
+import { Project, SyntaxKind } from 'ts-morph';
+
+export async function addNewComponentToInitComponents(
+  inputFilePath: string,
+  componentName: string
+) {
+  const code = await fs.promises.readFile(inputFilePath, 'utf8');
+
+  const project = new Project();
+  const sourceFile = project.createSourceFile(inputFilePath, code, {
+    overwrite: true,
+  });
+
+  const importStatementsDeclarations = sourceFile.getDescendantsOfKind(
+    SyntaxKind.ImportDeclaration
+  );
+
+  importStatementsDeclarations[0].addNamedImport(componentName);
```

Comment:
This in general is looking great. I think the last brittle piece could be this `[0]` here; if the imports change order, this would break. Is it simple enough to loop through the import declarations and match the one that's importing the components?

I don't want you to have to spend too much longer on this, so I'd also be happy starting with this and maybe opening a new issue to solidify things (and probably to convert the other pieces to use `ts-morph`, since this seems nicer).

Reply:
@daneah  I can give it a shot. I like what you mentioned, yes, I can improve it while I still have this PR opened.

Reply:
Update to nicer way in latest commit, so resolving this conversation.